script/os-autoinst-openvswitch

```diff
@@ -15,10 +15,12 @@ os-autoinst-openvswitch [OPTIONS]
 Show this help.
 
 =cut
+package main;
 
-package OVS;
```

Comment:
I know it's still a draft, but why are you renaming this to `main`? I think it would be good to keep it.

Reply:
I thought that since the name of the script incorporate a reference to openvswitch so calling the package OVS was too many repeats, but you are right [repetita iuvant](https://www.latin-is-simple.com/en/vocabulary/phrase/1653/), I'll update the package!